Source/Core/Window/WindowObject.cpp

```diff
@@ -45,7 +45,7 @@ WindowObject::WindowObject(WindowProperties properties)
 
 	glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
 	glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
-	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_COMPAT_PROFILE);
+	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
```

Comment:
Ai schimbat cu vreun motiv anume din COMPAT in CORE ?
Te intreb fiindca din cauza acestei modificari, cand rulezi Laborator2() se primeste eroare de GL_INVALID_OPERATION in Laborator2::CreateMesh() la apelul glVertexAttribPointer() atunci cand nu sunt facute TODO-urile de mai sus din functie (creare si bind VAO, VBO si IBO). Eroarea nu mai apare dupa ce sunt facute TODO-urile insa poate sa fie un pic confuz pentru studenti ca primesc eroare in consola (pentru cine se uita !).
Daca nu afecteaza cu nimic ce ai facut ulterior poti sa lasi pe COMPAT sau daca vrei neaparat pe CORE, adauga un comentariu in Laborator2::CreateMesh() deasupra apelului CheckOpenGLError() in care sa spui ca se va primi eroare pana se fac TODO-urile de creare si bind pt VAO, VBO si IBO.

Reply:
Nice catch! Am schimbat in `core` dintr-o serie de motive
- se elimina cu totul Fixed Function Pipeline — am intalnit de-a lungul timpului cazuri in care studentii foloseau `glBegin` si `glEnd`, de exemplu in realizarea temelor
- compatibilitate cu macOS
- am adaugat o precizare chiar in fereastra de debug

![image](https://user-images.githubusercontent.com/8483893/103650605-f5904d00-4f68-11eb-9670-fc3256c99c44.png)